backend.py

```diff
@@ -1717,7 +1713,7 @@ def isSWRAIDSyncd(primary_disk):
         logger.log("Failed to check if SWRAID device is sync'd due to: " + str(ex))
         return False
 
-def completeSWRAID(mounts, primary_disk, physical_disks, write_boot_entry, install_type, branding, boot_partnum, primary_partnum, logs_partnum, target_boot_mode):
+def completeSWRAID(mounts, primary_disk, physical_disks, write_boot_entry, install_type, branding, boot_partnum, primary_partnum, logs_partnum, target_boot_mode, remount=True):
```

Comment:
Having a boolean parameter to call `mountVolumes()` like this doesn't feel right. Why not just add `mountVolumes()` to the finalization sequence?

(Though perhaps some of this might change based on the discussion in #248.)